CONTRIBUTING.md

```diff
@@ -0,0 +1,15 @@
+# Contributing
+
+Initial setup
+
+```console
+npm install
+```
```

Comment:
@parkerbxyz based on your comment [here](https://github.com/actions/create-github-app-token/pull/217#issuecomment-2752048180), `console` seems to be useful for usage examples of the console built into node or web browsers. But `npm install` is something you'd run in shell/bash, shouldn't we use `shell` instead?